android-sdk/README.md

```diff
@@ -0,0 +1,80 @@
+MCP Android SDK Goals/Metrics
+
+# Cross Cutting Goals
+
+* Releases (mavencentral, jcenter, jitpack.io etc)
+    * Release final builds to to Maven Central
+    * Continue using [Sonatype](https://oss.sonatype.org) nexus for snapshots
+
+* CI/CD
+    * Setup auto publishing of SNAPSHOT builds
+
+* BOM style dependency management 
+    * Source a gradle plugin to support BOMs
+    * Ex [https://github.com/aerogear/aerogear-parent](https://github.com/aerogear/aerogear-parent)
+
+* Move everything to aerogear github organization
+  * What is included in everything?
+
+* Event driven API (either promise style or listeners. NO CALLBACKS! [They are hard to use in Android]
```

Comment:
Aren't listeners just a kind of callback?

Reply:
Yes and no.  There is a bit more formality with listeners usually than callbacks which are usually written as anonymous inner classes inline with a function reference.  

The comment is basically saying that I want function calls to lean more towards formality than toward inline callbacks.  I can whip up some pseudo code if you like to demonstrate my point more clearly.